release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -0,0 +1,1627 @@
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the table below. Additional details for more functionality that has been deprecated and removed are listed after the table.
+
+In the following tables, features are marked with the following statuses:
+
+* _Not Available_
+* _Technology Preview_
+* _General Availability_
+* _Deprecated_
+* _Removed_
+
+[discrete]
+[id="ocp-release-note-bare-metal-dep-rem_{context}"]
+=== Bare metal monitoring deprecated and removed features
+
+.Bare Metal Event Relay Operator tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Bare Metal Event Relay Operator
+|Removed
+|Removed
+|Removed
+|====
+
+[discrete]
+[id="ocp-release-note-images-dep-rem_{context}"]
+=== Images deprecated and removed features
+
+.Images deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Cluster Samples Operator
+|Deprecated
+|Deprecated
+|Deprecated
+|====
+
+[discrete]
+[id="ocp-release-note-install-dep-rem_{context}"]
+=== Installation deprecated and removed features
+
+.Installation deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|`--cloud` parameter for `oc adm release extract`
+|Deprecated
+|Deprecated
+|Deprecated
+
+|CoreDNS wildcard queries for the `cluster.local` domain
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`compute.platform.openstack.rootVolume.type` for {rh-openstack}
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`controlPlane.platform.openstack.rootVolume.type` for {rh-openstack}
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`ingressVIP` and `apiVIP` settings in the `install-config.yaml` file for installer-provisioned infrastructure clusters
+|Deprecated
+|Deprecated
+|Deprecated
+
+|Package-based {op-system-base} compute machines
+|Deprecated
+|Deprecated
+|Deprecated
+
+|Managing machines with the Cluster API for {azure-full}
+|Not Available
+|Technology Preview
+|Technology Preview
+
+|`platform.aws.preserveBootstrapIgnition` parameter for {aws-first}
+|Deprecated
+|Deprecated
+|Deprecated
+
+|Installing a cluster on {aws-short} with compute nodes in {aws-short} Outposts
+|Deprecated
+|Deprecated
+|Deprecated
+|====
+
+[discrete]
+=== Machine management deprecated and removed features
+
+.Machine management deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Managing machine with Machine API for {alibaba}
+|Removed
+|Removed
+|Removed
+
+|Cloud controller manager for {alibaba}
+|Removed
+|Removed
+|Removed
+
+|====
```

Comment:
These can finally be removed since it's been 3 releases (4.16+)
```suggestion
////
[discrete]
=== Machine management deprecated and removed features

.Machine management deprecated and removed tracker
[cols="4,1,1,1",options="header"]
|====
|Feature |4.17 |4.18 |4.19

|Placeholder
|Status
|Status
|Status

|====
////
```